Heroes.SDK.Library/Definitions/Structures/RenderWare/Variables.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Heroes.SDK.Definitions.Structures.RenderWare
+{
+    public unsafe struct RwV3d
```

Comment:
Consider replacing with `System.Numerics.Vector3`, or a wrapper around it. There's some special magic in the Runtime which makes operations on top of that be SIMD'ed when possible.